src/components/Link/Link.scss

```diff
@@ -39,22 +44,9 @@
   @include ms-Link;
 }
 
-// TODO: Remove overrides below.
```

Comment:
The only thing I left was the border: none; on the BUTTON.ms-link, however I just tried moving that up to the ms-link mixin as well and it works fine. Just added an update with the entire BUTTON.ms-link selector removed as it was not necessary. See screenshot below. The link on the right is the button:

![after](https://cloud.githubusercontent.com/assets/13757784/17532827/35f2cfa8-5e37-11e6-80ca-96a2a71599d9.png)